package.json

```diff
@@ -75,7 +77,10 @@
     "chai-http": "^4.0.0",
     "cheerio": "^1.0.0-rc.3",
     "eslint-config-oclif": "^3.0.0",
+    "expect": "^24.9.0",
```

Comment:
Future PR to remove chai (and code mod to `expect`)?

Reply:
Yes! Custom matchers in `expect` are so much easier to write, and you don't have to deal with weird Chai idiosyncrasies like `.deep.equal` or `.deep.contains` only performing partial matching at the top-level, or even just the confusing nature of "_which chai assertion should I use and which order should the `.confusing.extended.syntax.go`?_"